pkgs/desktops/plasma-5/srcs.nix

```diff
@@ -1,422 +1,422 @@
 # DO NOT EDIT! This file is generated automatically.
-# Command: ./maintainers/scripts/fetch-kde-qt.sh pkgs/desktops/plasma-5/
+# Command: ./maintainers/scripts/fetch-kde-qt.sh pkgs/desktops/plasma-5
```

Comment:
Is that supposed to change?

Reply:
I didn't touch the script and `srcs.nix` is 100% generated so I'll go with yes.

Reply:
Maybe it changes depending on the directory you supply.

Reply:
It does; the script doesn't really do any normalization of the directory name.